diff_cover/report_generator.py

```diff
@@ -219,6 +219,7 @@ def _src_path_stats(self, src_path):
     loader=TEMPLATE_LOADER,
     trim_blocks=True,
     lstrip_blocks=True,
+    autoescape=select_autoescape(),
```

Comment:
I think you just want `autoescape=True`

Reply:
Or import the function you are calling. But seems like setting the property true is mildly easier since we don't need to set any specific behavior

Reply:
:sigh: that awkward feeling when you comment from the wrong account. That is me though.

Reply:
@Bachmann1234 Made the change.